src/drivers/netconsole.c

```diff
@@ -0,0 +1,88 @@
+#include <kernel.h>
+#include <lwip.h>
+#include "console.h"
+#include "netconsole.h"
+
+typedef struct netconsole_driver {
+    struct console_driver c;
+    heap h;
+    struct udp_pcb *pcb;
+    ip_addr_t dst_ip;
+    u16 port;
+    boolean setup;
+} *netconsole_driver;
+
+#define MAX_PAYLOAD 1472
```

Comment:
What about just using something like 512--under the 576 minimum MTU for IP so guaranteed to be processable, especially since most lines will probably be less than that anyway.